autoload/sj/argparser/html_args.vim

```diff
@@ -16,6 +16,8 @@ function! sj#argparser#html_args#Process() dict
       endif
       call self.Next()
       continue
+    elseif self.body[0] =~ '{'
+      call self.JumpPair('{', '}')
```

Comment:
You can squeeze this into the condition below. So, it would be something like, `if self.body[0] =~ '["''{]' | call self.JumpPair('"''{', '"''}')`.